client/src/themes/theme.ts

```diff
@@ -1,6 +1,6 @@
-import { createMuiTheme } from '@material-ui/core';
+import { createMuiTheme, responsiveFontSizes } from '@material-ui/core';
 
-export const theme = createMuiTheme({
+let theme = createMuiTheme({
```

Comment:
You could keep this as a const, something like 
```suggestion
const baseTheme = createMuiTheme({
```

Then below

```
const theme = responsiveFontSizes(theme);
```